src/MessagingThreadListItem/DraftPencilIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+
+export function DraftPencilIcon() {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="12"
+      height="12"
+      fill="none"
+      viewBox="0 0 12 12"
+    >
+      <title>message draft icon</title>
```

Comment:
I believe this title takes care of screen-reader accessibility for the icon the same way as alt text on an img, right?

Reply:
Yep, that's my understanding!